### ISSUE TITLE
 Not working with Schibsted-account

### ISSUE THREAD

**Issue by markusludvigsson** [mathiazom/pasjonsfrukt#17]
Hello!

Was really excited to find this tool!

It seems however that the authentication part is no longer working after they are enforcing Schibsted-accounts for Podme?

Would be great if you make a fix for that (or clarify in the readme how to use the tool with it)

**Comment by bendikrb** [comment]
Ensure you are using the latest version, then try again :)

**Comment by markusludvigsson** [comment]
I just followed the documenation in https://github.com/mathiazom/pasjonsfrukt  - readme.md
"pip install pasjonsfrukt" should give me the latest version right?

**Comment by bendikrb** [comment]
> "pip install pasjonsfrukt" should give me the latest version right?

Yes, but try `pip install pasjonsfrukt==0.2.4` just to make sure.
And perhaps delete `~/.config/podme_api` if it exists..

**Comment by mathiazom** [comment]
Closing this as stale, feel free to open a new issue if needed.
